src/java/org/apache/cassandra/config/Config.java

```diff
@@ -814,6 +813,8 @@ public static void setClientMode(boolean clientMode)
     public volatile int items_per_collection_fail_threshold = DISABLED_GUARDRAIL;
     public volatile int fields_per_udt_warn_threshold = DISABLED_GUARDRAIL;
     public volatile int fields_per_udt_fail_threshold = DISABLED_GUARDRAIL;
+    public volatile int minimum_keyspace_rf_warn_threshold = DISABLED_GUARDRAIL;
+    public volatile int minimum_keyspace_rf_fail_threshold = DISABLED_GUARDRAIL;
```

Comment:
I think replication factors are always for keyspaces, so I wonder if this would be better named just `minimum_replication_factor_fail_threshold`, or perhaps `min_replication_factor_fail_threshold`.

Reply:
I am cool with all 3 names, slight preference for `minimum_replication_factor_fail_threshold`

Reply:
I'd prefer either `minimum_replication_factor_fail_threshold` or `min_replication_factor_fail_threshold `. I guess that the prefix choice that we make now would determine how future min/max (or minimum/maximum) properties would look like, since new similar properties should use the same prefix.